spec/helpers/script_helper_spec.rb

```diff
@@ -40,13 +40,38 @@
         output = render_javascript_pack_once_tags
 
         expect(output).to have_css(
-          "script[src^='/polyfill.js'][nomodule] ~ \
-          script[src^='/application.js'] ~ \
-          script[src^='/document-capture.js']",
+          "script:not([crossorigin])[src^='/polyfill.js'][nomodule] ~ \
+          script:not([crossorigin])[src^='/application.js'] ~ \
+          script:not([crossorigin])[src^='/document-capture.js']",
           count: 1,
           visible: :all,
         )
       end
+
+      context 'local development crossorigin sources' do
+        before do
+          allow(Rails.env).to receive(:development?).and_return(true)
+        end
+
+        around do |example|
+          original_webpack_port = ENV['WEBPACK_PORT']
+          ENV['WEBPACK_PORT'] = '3000'
+          example.run
+          ENV['WEBPACK_PORT'] = original_webpack_port
+        end
```

Comment:
I'd welcome any feedback if there's a more elegant way to stub this.

Reply:
💡 
```suggestion
        before { stub_const('ENV', 'WEBPACK_PORT' => '3000') }
```